fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -49,7 +49,8 @@ def load_data(self, X, y, event=None, **kwargs):
         self.index = signed.index
         self.default_objective_lambda_vec = None
 
-        event_vals = self.tags[_EVENT].unique()
+        # fill in the information about the basis
+        event_vals = self.tags[_EVENT].dropna().unique()
```

Comment:
I thought when we chatted earlier today you said `dropna()` isn't needed anymore? Perhaps I misunderstood :-)

Reply:
The `pd.dropna` would not be required in the case of `pd.groupby`. But over here, if the `pd.dropna` is not present, it will take `nan` also as a unique value.

Reply:
Got it.